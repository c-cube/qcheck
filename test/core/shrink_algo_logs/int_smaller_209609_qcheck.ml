open QCheck

(* test from issue #59 *)
let ints_smaller_209609 =
  Test.make ~name:"ints < 209609"
    (small_int_corners()) (Log.shrinks_int @@ fun i -> i < 209609)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand ints_smaller_209609 with Test.Test_fail _ -> ()
