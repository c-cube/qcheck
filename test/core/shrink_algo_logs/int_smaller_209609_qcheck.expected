holds 0
holds 1
holds 2
fails 4611686018427387903
holds 0
fails 2305843009213693952
holds 0
fails 1152921504606846976
holds 0
fails 576460752303423488
holds 0
fails 288230376151711744
holds 0
fails 144115188075855872
holds 0
fails 72057594037927936
holds 0
fails 36028797018963968
holds 0
fails 18014398509481984
holds 0
fails 9007199254740992
holds 0
fails 4503599627370496
holds 0
fails 2251799813685248
holds 0
fails 1125899906842624
holds 0
fails 562949953421312
holds 0
fails 281474976710656
holds 0
fails 140737488355328
holds 0
fails 70368744177664
holds 0
fails 35184372088832
holds 0
fails 17592186044416
holds 0
fails 8796093022208
holds 0
fails 4398046511104
holds 0
fails 2199023255552
holds 0
fails 1099511627776
holds 0
fails 549755813888
holds 0
fails 274877906944
holds 0
fails 137438953472
holds 0
fails 68719476736
holds 0
fails 34359738368
holds 0
fails 17179869184
holds 0
fails 8589934592
holds 0
fails 4294967296
holds 0
fails 2147483648
holds 0
fails 1073741824
holds 0
fails 536870912
holds 0
fails 268435456
holds 0
fails 134217728
holds 0
fails 67108864
holds 0
fails 33554432
holds 0
fails 16777216
holds 0
fails 8388608
holds 0
fails 4194304
holds 0
fails 2097152
holds 0
fails 1048576
holds 0
fails 524288
holds 0
fails 262144
holds 0
holds 131072
holds 196608
fails 229376
holds 0
holds 114688
holds 172032
holds 200704
fails 215040
holds 0
holds 107520
holds 161280
holds 188160
holds 201600
holds 208320
fails 211680
holds 0
holds 105840
holds 158760
holds 185220
holds 198450
holds 205065
holds 208373
fails 210027
holds 0
holds 105014
holds 157521
holds 183774
holds 196901
holds 203464
holds 206746
holds 208387
holds 209207
fails 209617
holds 0
holds 104809
holds 157213
holds 183415
holds 196516
holds 203067
holds 206342
holds 207980
holds 208799
holds 209208
holds 209413
holds 209515
holds 209566
holds 209592
holds 209605
fails 209611
holds 0
holds 104806
holds 157209
holds 183410
holds 196511
holds 203061
holds 206336
holds 207974
holds 208793
holds 209202
holds 209407
holds 209509
holds 209560
holds 209586
holds 209599
holds 209605
holds 209608
fails 209610
holds 0
holds 104805
holds 157208
holds 183409
holds 196510
holds 203060
holds 206335
holds 207973
holds 208792
holds 209201
holds 209406
holds 209508
holds 209559
holds 209585
holds 209598
holds 209604
holds 209607
fails 209609
holds 0
holds 104805
holds 157207
holds 183408
holds 196509
holds 203059
holds 206334
holds 207972
holds 208791
holds 209200
holds 209405
holds 209507
holds 209558
holds 209584
holds 209597
holds 209603
holds 209606
holds 209608
